datahub-web-react/src/app/shared/tags/AddTagsTermsModal.tsx

```diff
@@ -210,6 +211,7 @@ export default function EditTagTermsModal({
             if (isValidTagName(inputValue)) {
                 setShowCreateModal(true);
             }
+            setInputValue('');
```

Comment:
and tests are failing because of this line I believe.

If you clear the input value and save the new term, we submit an empty string to the backend on creating the tag.

You can test by trying to create a new tag with this code.